internal/config/default/config.toml

```diff
@@ -7,7 +7,7 @@ limit = 0
   jump_to_parent = ["J"]
   jump_to_children = ["K"]
   jump_to_working_copy = ["@"]
-  apply = ["enter"]
+  apply = ["enter", "alt+enter"]
```

Comment:
I think this should be a separate key binding. The reason for that is that we want user to press it willingly. I know later on in the code you are checking if the `Alt` modifier is active but then if the user changes this key bindings to something else that doesn't have `alt` in it that it won't work.